test/adapters/substrate/adapter/wasm-adapter/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
this is out of its place as it has nothing to do with substrate. All hosts should support wasm. So maybe we need test/testers or test/test_helpers to keep the universal bit of testing machine.

Reply:
I agree that the name `wasm-adapter` is confusing. I renamed it to `substrate-adapter-runtime`.  After all it is adapter specific, as only the substrate adapter does not call the Host API directly but through a thin wasm wrapper.

Reply:
we don't test host api by calling them directly. we test them by running the wasm. So it is not an objection to the wasm-adapter name but the place of the adapter being in substrate subfolder. Could you add this also to #191 subtasks?